### DISCUSSION TITLE
Establishing connection to an other user

### DISCUSSION THREAD

**Discussion by YoussefHachicha** [shepeliev/webrtc-kmp#117] [General]
Hello!
Thank you for this awesome library!

but how can I establish a connection to an other user the docs don't specify how to call an other user, could you provide some more docs on how to do that, that would be very helpfull

**Comment by shepeliev** [comment]
Hello @YoussefHachicha,

Establishing connection between users is out of scope of this library and WebRTC in common. I think this [article](https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/Signaling_and_video_calling) may be usefull.

**Reply by YoussefHachicha** [reply]
thanks for the article i have read it but still WebRtc is a big part of achieving this, the only thing that this library need is a Websocket or a socket Io right ?
